packages/cli/src/commands/generate/component/templates/component.tsx.template

```diff
@@ -1,4 +1,4 @@
-const ${pascalName} = () => {
+const ${pascalName}: React.FunctionComponent = () => {
```

Comment:
Do we want to proactively type generated components, in case people want to use FC props like `children`?

Reply:
I think that would be awesome